api-definitions/db_positions/api.yaml

```diff
@@ -335,27 +375,35 @@ definitions:
     properties:
       id:
         $ref: "#/definitions/PaymentOptionId"
+      creditor_reference_id:
+        $ref: "#/definitions/CreditorReferenceId"
       amount:
         $ref: "#/definitions/Amount"
       due_date:
         $ref: "#/definitions/Timestamp"
+      retention_date:
+        $ref: "#/definitions/Timestamp"
       is_conclusive:
         type: boolean
         default: true
-      payment_description: 
+      metadata:
         type: string
-        minLength: 1
-        example: TARI 2021 soluzione unica
+        maxLength: 3000
+        description: 
+          type: string
+          maxLength: 3000
+          Optional field for metadata (a max of 10 key-value pairs)
```

Comment:
Could you define `metadata` as array type with `minItems ` and `maxItems ` properties ?
I this that description it isn't helpful

Reply:
Ok, the type was corrected with your suggestions.